src/Nethermind/Nethermind.Runner/JsonRpc/Startup.cs

```diff
@@ -47,8 +47,9 @@ public class Startup
 
         public void ConfigureServices(IServiceCollection services)
         {
-            // ReSharper disable once ASP0000
+            #pragma warning disable ASP0000
             ServiceProvider sp = services.BuildServiceProvider()!;
+            #pragma warning restore ASP0000
```

Comment:
ReSharper disables only affect resharper, while `#pragma warning ...` should work for all compilers.